misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -223,3 +227,17 @@ data class CorsConfig(
   /** A comma separated list of HTTP headers that are allowed to be exposed on the client. */
   val exposedHeaders: Array<String> = arrayOf()
 )
+
+data class ConcurrencyLimiterConfig(
+  /** If true, disables automatic load shedding when degraded. */
```

Comment:
this comment describes this as a "disabled" variable, maybe should be: `If false, disables automatic load shedding`?